charts/mlrun-ce/admin_installation_values.yaml

```diff
@@ -0,0 +1,397 @@
+# Use this Values file for installing MLRun CE cluster resources such as: crd, cluster role on a namespace that will not use for
```

Comment:
these 3 new value files have code dup, helm support installing chart with multiple helm values, overriding each other. instead of code duplicating the values, use the overriding values only so each file will have its needed values only.